### PR TITLE
Adding a basic experimentation playground to the repo

### PR DESCRIPTION
**Pull request by Dan503**
It's been a massive pain trying to test whatever the current build of Autoprefixer is while it's been under development.

I've been resorting to opening the Autoprefixer repo, compiling it, copying the compiled assets into the node_modules folder of a completely unrelated project and then running the compiler for that project to get output CSS.

**That is an insane workflow!**

There needed to be an easier way to play around with the current Autoprefixer build and see if things are working as they are supposed to without having to go to all the trouble of creating formal tests.

So with that in mind I set up this simple little experimentation playground inside the Autoprefixer repo.

1. Run the command `gulp play`
2. Write some CSS in `playground.input.css` and save the file
3. View the output CSS in `playground.output.css`

For some reason it isn't translating CSS Grid code even though I've turned the setting on. @ai Do you have any idea why it isn't outputting CSS Grid code?

### CODE REVIEW

File: playground/playground.output.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+/*
+  This file is purely just for experimentation.
+  1. Run the command "gulp play"
+  2. Write some CSS in playground.input.css and save the file
+  3. View the output CSS in playground.output.css
+*/
+
+.grid {
+  display: grid;
+}
```
Comment (Dan503) [inline playground/playground.output.css]: This should be outputting:

```css
.grid {
  display: -ms-grid;
  display: grid;
}
```

**Comment by Dan503** [comment]
> 97:30error  "./build" is not foundnode/no-missing-require

Ummm... that is not possible to fix. I've purposely done it that way so that it is running off of the real currently active code.

I purposely did not require it until I could guarrantee that the folder exists.

Can we disable that rule or tell Travis CLI to ignore that line?

**Comment by Dan503** [comment]
[This page on Stack Overflow](https://stackoverflow.com/a/48028726/1611058) says that you can ignore a line of code using this

```
// @codingStandardsIgnoreLine
```

It didn't seem to work though 😥

**Comment by Dan503** [comment]
The alternative is checking the "build" folder into git so that it exists.

**Comment by ai** [comment]
Do not forget to check that `build/` will not contain `playground` after `gulp build`.

**Comment by ai** [comment]
> 97:30error "./build" is not foundnode/no-missing-require

Ignore it (you can fix it by calling `gulp build`)

**Comment by Dan503** [comment]
Ummm... by merging this as it is though, Travis CI will fail for all future pull requests.

I think the easiest way to resolve this is to save build into git.

**Comment by Dan503** [comment]
It's also not outputting grid translations like it's supposed to.

**Comment by ai** [comment]
> I think the easiest way to resolve this is to save build into git.

Nope, generated files should be out of the git.

I just disabled that rule. Anyway it brings me problems in different places.

There is no Grid because `browserslist` in `package.json` has no IE.

**Comment by Dan503** [comment]
Ok, would you prefer the IE support be applied to browserlist in package.json or in the gulp task?

The gulp task will keep it specific to just the playground.

I don't understand why you are excluding IE by default in the browser list 😕

**Comment by ai** [comment]
> I don't understand why you are excluding IE by default in the browser list 😕

I added this Browserslist for Babel. I didn't found when users will need to run Autoprefixer JS files in IE.

> The gulp task will keep it specific to just the playground.

The best way is to put additional Browserslist to `playground/.browserslistrc` with `defaults` query. It will be explicit to users what browsers are used there.

**Comment by Dan503** [comment]
Ok, I'll do that tonight after work 😊👍

**Comment by Dan503** [comment]
> Do not forget to check that `build/` will not contain `playground` after `gulp build`.

Playground doesn't doesn't live in the build folder. It will never be inside that folder.

**Comment by ai** [comment]
> Playground doesn't doesn't live in the build folder. It will never be inside that folder.

Nope, build scripts copy everything to `build/`. But don't worry, I added playground to ignore list.

**Comment by Dan503** [comment]
Don't ignore the whole playground folder. You at least need playground.input.css to stay saved in Git.

**Comment by Dan503** [comment]
playground.output.css is a generated file so it's understandable if you want to ignore that one.

**Pull request merged**